src/assets/scss/components/tab-nav.scss

```diff
@@ -1,16 +1,47 @@
-@import '../../../variables';
-@import 'src/geometry.scss';
+@import 'src/assets/scss/functions';
 
-.nav-tab {
+.alg-nav-tab {
   display: flex;
-  font-family: inherit;
-  overflow: hidden;
+  align-items: center;
+  position: relative;
 
-  .nav-tab-item {
-    @include main-tabs;
```

Comment:
don't forget to plan the cleanup when everything is done

Reply:
Fixed